pallets/polkadex-ido/src/lib.rs

```diff
@@ -101,11 +101,11 @@ pub mod weights;
 use pallet_polkadex_ido_primitives::{FundingRoundWithPrimitives, StringAssetId, VoteStat};
 pub use weights::WeightInfo;
 
-#[cfg(test)]
-mod mock;
-
-#[cfg(test)]
-mod test;
+// #[cfg(test)]
+// mod mock;
+//
+// #[cfg(test)]
+// mod test;
```

Comment:
Why is this commented out?

Reply:
IDO and faucet will be removed from develop